src/plugins/intel_gpu/tests/test_cases/activation_simple_gpu_test.cpp

```diff
@@ -1747,11 +1744,39 @@ const std::vector<tensor> inputShapes = {
     {16, 16, 5, 5},
 };
 
-const auto fpFunctionsParams = ::testing::Combine(::testing::ValuesIn(dataTypes),
-                                                  ::testing::ValuesIn(types),
-                                                  ::testing::ValuesIn(inputShapes),
-                                                  ::testing::ValuesIn(activationFunctions),
-                                                  ::testing::Values(activation_additional_params{}),
-                                                  ::testing::Values(padding{}));
-
-INSTANTIATE_TEST_SUITE_P(fp_activation_blocked_tests, activation_random_test, fpFunctionsParams);
+INSTANTIATE_TEST_SUITE_P(
+    fp_activation_blocked_tests0,
+    activation_random_test,
+    ::testing::Combine(::testing::Values(dataTypes[0]),
+                       ::testing::Values(types[0]),
+                       ::testing::Values(inputShapes[0]),
+                       ::testing::ValuesIn(activationFunctions),
+                       ::testing::Values(activation_additional_params{}),
+                       ::testing::Values(padding{})));
+INSTANTIATE_TEST_SUITE_P(
+    fp_activation_blocked_tests1,
+    activation_random_test,
+    ::testing::Combine(::testing::Values(dataTypes[1]),
+                       ::testing::Values(types[1]),
+                       ::testing::Values(inputShapes[1]),
+                       ::testing::ValuesIn(activationFunctions),
+                       ::testing::Values(activation_additional_params{}),
+                       ::testing::Values(padding{})));
+INSTANTIATE_TEST_SUITE_P(
+    fp_activation_blocked_tests2,
+    activation_random_test,
+    ::testing::Combine(::testing::Values(dataTypes[0]),
+                       ::testing::Values(types[2]),
+                       ::testing::Values(inputShapes[2]),
+                       ::testing::ValuesIn(activationFunctions),
+                       ::testing::Values(activation_additional_params{}),
+                       ::testing::Values(padding{})));
+INSTANTIATE_TEST_SUITE_P(
+    fp_activation_blocked_tests3,
+    activation_random_test,
+    ::testing::Combine(::testing::Values(dataTypes[1]),
+                       ::testing::Values(types[3]),
+                       ::testing::Values(inputShapes[2]),
```

Comment:
sorry for the belated review, but I'm afraid this will break when the number of elements in types or dataTypes change. Could you make it more general?
for example)
   dataTypes[0] * activationFunctions
   dataTypes * activationFunctions[0]

Reply:
Done.